data/language/en-GB.txt

```diff
@@ -3438,6 +3438,7 @@ STR_6331    :Create ducks
 STR_6332    :Remove ducks
 STR_6333    :Increase scale factor
 STR_6334    :Decrease scale factor
+STR_6335    :Tile Inspector: Toggle wall slope
```

Comment:
Not sure we reuse string IDs or always put new ones at the end. @OpenRCT2/developers ?

Reply:
We normally don't re-use them, this should be added to the end indeed.

Reply:
Resolved, thank you. :)